File: BAEKJOON/11866.js
const fs = require('fs');
const filePath = process.platform === 'linux' ? '/dev/stdin' : __dirname + '/input.txt';
const input = fs.readFileSync(filePath).toString().trim().split(' ').map(Number);

function solution([humanNum, deleteNum]) {
  const humanArr = new Array(humanNum);
  const answer = new Array();
  let count = 0;

  for (let i = 0; i < humanNum; i++) {
    humanArr[i] = i + 1;
  }

  while (humanArr.length) {
    let firstHuman = humanArr.shift();
    count++;

    if (count % deleteNum === 0) {
      answer.push(firstHuman);
    } else {
      humanArr.push(firstHuman);
    }
  }

  console.log(`<${answer.join(', ')}>`);
}

solution(input);

File: baekjoon/11866.py
from collections import deque
n, k = map(int, input().split())
nQueue = deque([i for i in range(1, n + 1)])
answer = []

while len(nQueue):
    for count in range(k - 1):
        nQueue.rotate(-1)  # (k - 1)번 큐를 회전시킨다.
    answer.append(nQueue.popleft()) # k번째에서 사람을 제거한다.

print(f"<{', '.join(map(str, answer))}>")